src/main/java/org/folio/circulation/domain/Loan.java

```diff
@@ -210,10 +211,11 @@ public String getAction() {
     return getProperty(representation, ACTION);
   }
 
-  public Loan changeStatusOfUsageAtLocation(String usageStatus) {
-    log.debug("changeStatusOfUsageAtLocation:: parameters usageStatus: {}", usageStatus);
+  public Loan changeStatusOfUsageAtLocation(String usageStatus, ZonedDateTime holdShelfExpirationDate) {
```

Comment:
Let's keep the method `changeStatusOfUsageAtLocation(String usageStatus)` in this class. It will `return changeStatusOfUsageAtLocation(usageStatus, null)`, this way there will be no need in multiple changes in other parts of the code.